pkg/surveyext/editor.go

```diff
@@ -104,7 +104,9 @@ func (e *GhEditor) prompt(initialValue string, config *survey.PromptConfig) (int
 			break
 		}
 		if r == '\r' || r == '\n' {
-			if e.BlankAllowed {
```

Comment:
It also might make sense to rename BlankAllowed to something like SkipEditorAllowed, since skipping no longer just returns an empty string.

Reply:
Makes sense to me too! Thanks for looking into this

Reply:
Sure!  This is taken care of in commit 6497114a864da1ddb9a52031d67d7bbba523bf10.